articles/firewall/firewall-best-practices.md

```diff
@@ -17,7 +17,7 @@ To maximize the [performance](firewall-performance.md) of your Azure Firewall an
 
 - **Exceeding rule limitations**
 
-   If you exceed limitations, such as using over 20,000 unique source/destination combinations in rules, it can affect firewall traffic processing and cause latency. Even though this is a soft limit, if you surpass this value it can affect overall firewall performance. For more information, see the [documented limits](../nat-gateway/tutorial-hub-spoke-nat-firewall.md).
+   If you exceed limitations, such as using over 20,000 unique source/destination combinations in rules, it can affect firewall traffic processing and cause latency. Even though this is a soft limit, if you surpass this value it can affect overall firewall performance. For more information, see the [documented limits](../../includes/firewall-limits.md).
```

Comment:
```suggestion
   If you exceed limitations, such as using over 20,000 unique source/destination combinations in rules, it can affect firewall traffic processing and cause latency. Even though this is a soft limit, if you surpass this value it can affect overall firewall performance. For more information, see the [documented limits](../azure-resource-manager/management/azure-subscription-service-limits.md#azure-firewall-limits).
```
@vhorne - includes in the /includes/ folder aren't build (rendered) by themselves. They're only available for viewing on the site when they're inside an article. You can add the include into this page or link to another page that already contains the include, but you can't link directly to the include file.   I've provided a jump link to the firewall limits on the Azure subscription service limits page: https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#azure-firewall-limits